lld/ELF/Relocations.cpp

```diff
@@ -1337,7 +1363,9 @@ unsigned RelocationScanner::handleTlsRelocation(RelExpr expr, RelType type,
     // set NEEDS_TLSDESC on the label.
     if (expr != R_TLSDESC_CALL) {
       if (!isRISCV || type == R_RISCV_TLSDESC_HI20)
```

Comment:
```
if (aarch64) ...;
else if (!isRISCV || type == ...) ...;
```

Reply:
Fixed in 5f2ee82947af1039b5c4cd4347cfefea7c3ba967, thanks